src/khoj/processor/github/github_to_jsonl.py

```diff
@@ -39,17 +39,13 @@ def wait_for_rate_limit_reset(response, func, *args, **kwargs):
             return
 
     def process(self, previous_entries=None):
-        # If demo mode is enabled, don't re-process any of the repositories. This is resource intensive.
-        if state.demo and previous_entries is not None:
-            return self.update_entries_with_ids(previous_entries, previous_entries)
-
```

Comment:
Don't we want to disable github (re-)indexing in demo mode?

Reply:
No, I don't think so. I've set it up so that the periodic scheduler should not be enabled in `demo` mode, and I think that should be sufficient. We should have a backup mechanism to trigger a re-indexing if required on the live app.

Reply:
Ah I see. Makes perfect sense